server/src/db/-- SQLite.sql

```diff
@@ -0,0 +1,36 @@
+-- SQLite
+DROP TABLE IF EXISTS Ticket;
+DROP TABLE IF EXISTS TicketHistory;
+
+CREATE TABLE Ticket(
+    TicketID INTEGER PRIMARY KEY,
+    ServiceID INTEGER REFERENCES Service(ServiceID) ON DELETE NO ACTION ON UPDATE CASCADE,
+    IssuedTime DATE DEFAULT NULL,
```

Comment:
You may want to use the default value in the database. In this case, you don't need any computation in your code.
I mean:
IssuedTime TEXT DEFAULT CURRENT_TIMESTAMP

Reply:
I didn't bother recalling all the defaults, if it gives the date in the YYYY-MM-DD format then yes it can be used to avoid computation in the code.

Reply:
It was just a suggestion. There is no must.
It's all good.